src/LayoutManager/LayoutManager.spec.ts

```diff
@@ -858,7 +858,7 @@ describe('Layout Manager', () => {
       });
       expect(group).toMatchObject({ width: 100, height: 300 });
       expect(child.getCenterPoint()).toMatchObject({ x: 100, y: 100 });
-      expect(group.getCenterPoint()).toMatchObject({ x: 50, y: 150 });
+      expect(group.getCenterPoint()).toMatchObject({ x: 100, y: 100 });
```

Comment:
this is wrong

Reply:
You pass width and height and a fixed layout and no left and top
So the group's tl should be positioned at 0, 0
Anything is is not fixed layout

Reply:
i updated the readme since there was no explanation.
The group should not move the object that is wrapping unless a position is passed.
This is how it was before even the group rewrite.
If you artificially crafted a group that had smaller bbox than the objects, the bounding box would be centered around the group center cutting equally from all the sides.

The same issue here with origin, if you specify for this group center/center, it won't be positioned at 0,0 and also the object will be misplaced.